src/components/Layout/Card/section-card.tsx

```diff
@@ -1,109 +1,164 @@
 import Link from "next/link";
-import { Card } from ".";
 import { getVideos } from "lib/videos";
-import { useState } from "react";
+import { useContext, useState } from "react";
 import Image from "next/image";
-import full from "public/fullsendpodcast.jpeg"
-import david from "public/david.jpeg"
+import full from "public/fullSendHd.jpg"
+import david from "public/davidIcon.jpeg"
 import charles from "public/charles.jpeg"
 import mma from "public/mma.png"
-
+import ted from "public/ted.jpeg"
+import rogan from "public/rogan.jpg"
+import { CreatorContext } from "src/context";
+import rich from "public/rich.jpeg"
+import lex from "public/lex.jpeg"
 
 export const CardSection = () => {
-
-  const [listOpen, setListOpen] = useState(false);
+  const context = useContext(CreatorContext);
   const [selectCreator, setSelectCreator] = useState("creatorOne");
-  const [creatorTitle, setCreatorTitle] = useState("Full Send Podcast");
+ 
+
   let creatorData = getVideos(selectCreator);
 
+  return (
+    <>
+      <div className="">
+        <div className="flex flex-col md:flex-row">
+          <div className="overflow-x-hidden overflow-y-hidden w-screen flex flex-row">
+            <div className="w-96" onClick={() => {
+              setSelectCreator("creatorOne");
+              context.setCreator("Full Send Podcast");
+              context.setPayment("https://buy.stripe.com/test_9AQbLbdhD8Y5eYM3cf")
+              context.setImage("https://yt3.googleusercontent.com/5_TYQJ-59yU45NoK1GpQcRuov8OgZuwuSiS-0X8IveZI3QK_tKaiQxx9BHGqVkGogD08zY-txA=s900-c-k-c0x00ffffff-no-rj")
+            }}>
+              <Link href={`/video/PXPoKLPiSBM`} className="flex items-center px-4 py-2 dark:hover:bg-gray-600 dark:hover:text-white" >
+                <div className="relative transition ease-in-out delay-100 hover:-translate-y-1 hover:scale-110  duration-400">
+                  <Image width={"400"} height={"200"} className="mr-2 " src={full} alt="Jese image" />
+                  <div className="absolute bottom-1 bg-black opacity-75 w-full text-white font-semibold text-base lg:text-3xl">
+                    <span className="bottom-4 md:ml-4 ">Full Send</span>
+                  </div>
+                </div>
+              </Link>
+            </div>
+            <div className="w-96" onClick={() => {
+              setSelectCreator("creatorTwo");
+              context.setCreator("David Dobrik");
+              context.setPayment("https://buy.stripe.com/test_9AQbLb4L7del4k89AC")
+              context.setImage("https://yt3.googleusercontent.com/ytc/AL5GRJULlbEP4MO2qjUSF3X8op4Errek_FlWpoXfbZfRmA=s176-c-k-c0x00ffffff-no-rj")
```

Comment:
So many repetition 

Can you make one function 
The instead of repeating code

Reply:
Will do.